ui/src/App.tsx

```diff
@@ -12,7 +12,7 @@ function App() {
       <Layout>
         <Switch>
           <Route exact path="/" component={HomeScreen} />
-          <Route path="/posts" component={PostsScreen} />
+          <Route path="/:category(jobs|talents)/:id?" component={PostsScreen} />
```

Comment:
what will the `talents` route do?

Reply:
Should we split it to `/talents` and manage the talents in a separate `TalentsScreen`?

Reply:
ahh, so `talents` are people looking for jobs? If so it's fine to have it as it is as the UI is not going to change much.